akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -674,6 +725,14 @@ object Logging {
       case InfoLevel    ⇒ Info(logSource, logClass, message, mdc)
       case DebugLevel   ⇒ Debug(logSource, logClass, message, mdc)
     }
+
+    def apply(level: LogLevel, logSource: String, logClass: Class[_], message: Any, mdc: MDC, marker: LogMarker): LogEvent = level match {
+      case ErrorLevel   ⇒ Error(logSource, logClass, message, mdc, marker)
+      case WarningLevel ⇒ Warning(logSource, logClass, message, mdc, marker)
+      case InfoLevel    ⇒ Info(logSource, logClass, message, mdc, marker)
+      case DebugLevel   ⇒ Debug(logSource, logClass, message, mdc, marker)
+    }
+
   }
```

Comment:
you mean the `  class DefaultLogger extends Actor with StdOutLogger with RequiresMessageQueue[LoggerMessageQueueSemantics] {
`?

It prints the events as they come (raw `      case event: LogEvent     ⇒ print(event)`) so I made the to String include the marker (if it's an Info3 etc)